listenbrainz/tests/integration/test_feedback_api.py

```diff
@@ -1162,6 +1162,123 @@ def test_get_feedback_for_recordings_for_user_invalid_recording(self):
                                    query_string={"recording_mbids": recordings})  # recording_mbids has invalid recording_msid
         self.assert400(response)
         self.assertEqual(response.json["code"], 400)
+    # Started here
+    def test_get_feedback_for_recordings_for_post_method_for_user_valid_mbids(self):
+        inserted_rows = self.insert_test_data_with_mbid(self.user["id"])
+
+        # recording_msids for which feedback records are inserted
+        recordings = inserted_rows[0]["recording_msid"] + "," + inserted_rows[1]["recording_msid"]
```

Comment:
These should be arrays and not comma separated string in case of POST.

Reply:
Same comment for all other newly added tests.

Reply:
Resolved